### ISSUE TITLE
Please review comments on VnV Plan

### ISSUE THREAD

**Issue by smiths** [slimpotatoes/STEM_Moire_GPA#21]
Comments in commit 4358e18.

**Comment by slimpotatoes** [comment]
Thanks for the feedback @smiths . I modified the document following all the comments. 

I just let one task for later which is the definition and the generation of the tests dataset that are going to be used in the tests functions. When writing the test plan, I needed first to describe the aspects of the software I wanted to test in a manner that speaks to me. It's very mathy and not detailed enough to be directly implemented. Inputs and expected outputs in the system tests need to be more specific and adapted to the format used by the software. Those decisions were not taken because I was not sure if the test functions I proposed were appropriate. Now with the layout kind of set, I'm ready to fix the design of the test dataset. I will do it whenever I find some time :).

Again thanks for the feedback.
